apps/challenges/migrations/0088_add_environment_error_entry.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 2.2.20 on 2022-10-14 02:42
+
+import base.utils
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('challenges', '0087_add_cpu_only_code_upload_job_configs'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='challengeconfiguration',
+            name='code_upload_env_output_file',
+            field=models.FileField(blank=True, null=True, upload_to=base.utils.RandomFileName('zip_configuration_files/challenge_zip')),
+        ),
+        migrations.AlterField(
+            model_name='challenge',
+            name='worker_cpu_cores',
+            field=models.IntegerField(blank=True, default=512, null=True),
+        ),
+        migrations.AlterField(
+            model_name='challenge',
+            name='worker_memory',
+            field=models.IntegerField(blank=True, default=1024, null=True),
+        ),
+    ]
```

Comment:
@matthew-so why are there 3 migration files? Can you just delete these 3 once and regenerate them? If you are changing two model files there should only be two migration files